_sass/components/_buttons.scss

```diff
@@ -22,4 +23,104 @@
     //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
     text-decoration: none;
   }
+  &:active {
+    color: #ffffff;
+    background-color: lighten($primary, 50%);
+    transform: translateY(-1px);
+    //box-shadow: 0 7px 14px rgba(50, 50, 93, 0.1), 0 3px 6px rgba(0, 0, 0, 0.08);
```

Comment:
```suggestion
```

Reply:
Removed this whole block. Testing code not intended to be submitted.